docs/API.TraversingOutboundAndInboundLinksWithXgAndYg.md

```diff
@@ -54,45 +215,49 @@ If you wanted to retrieve both immediate parents and ultimate parents, you could
 
-### The Outbound Link Operator ('>') ###
+<a name="hdr7"></a>
+#### The Outbound Link Operator ('>') ####
 
-Suppose you have defined an xg value that returns a certain set of outbound links. You can then further expand this set to refer to higher levels of outbound links, using the '>' operator.
+Suppose you have defined an **xg** value that returns a certain set of outbound links. You can then further expand this set to refer to higher levels of outbound links, using the '>' operator.
 
     xg=<outboundExpansion> > <fieldIdentifier>
 
 For example:
 
     <cm-well-host>/permid.org?op=search&qp=CommonName.mdaas:Marriott%20Ownership,organizationCity.mdaas:Orlando&format=ttl&with-data&xg=hasImmediateParent.mdaas>hasUltimateParent.mdaas
 
-This query retrieves the outbound links in the hasImmediateParent fields of the matched infotons, and in addition, the outbound links in the hasUltimateParent fields of the level 1 outbound links.
+This query retrieves the outbound links in the **hasImmediateParent** fields of the matched infotons, and in addition, the outbound links in the **hasUltimateParent** fields of the level 1 outbound links.
 
 You can add a "chain" of several outbound link operators with field identifiers, to retrieve more and more remote links.
 
-### The Field Wildcard ###
+<a name="hdr8"></a>
+#### The Field Wildcard ####
 
 Instead of adding a specific field name after the '>' operator, you can use the wildcard character '_' to indicate all fields.
 
 For example:
 
     <cm-well-host>/permid.org?op=search&qp=CommonName.mdaas:Marriott%20Ownership,organizationCity.mdaas:Orlando&format=ttl&with-data&xg=hasImmediateParent.mdaas>_
 
-This query retrieves the outbound links in the hasImmediateParent fields of the matched infotons, and in addition, the outbound links *all* fields of the level 1 outbound links.
+This query retrieves the outbound links in the **hasImmediateParent** fields of the matched infotons, and in addition, the outbound links *all* fields of the level 1 outbound links.
 
-### Field Filters ###
+<a name="hdr9"></a>
+#### Field Filters ####
 
 You can add field filters to constrain the additional infotons retrieved by the **xg** flag. You do this by adding the field filters in [ ] brackets after the expansion definition.
 
     xg=<outboundExpansion>[field filter]
 
-The syntax of the field filter is the same as for the [qp parameter](API.FieldConditionSyntax.md), except that only the partial match and exact match operators (: and ::) are supported for the **xg** flag..
+The syntax of the field filter is the same as for the [qp parameter](API.FieldConditionSyntax.md), except that only the partial match and exact match operators (: and ::) are supported for the **xg** flag. (The fuzzy match operator ~ is not supported for **xg**, because this would produce a prohibitively expensive query.)
```

Comment:
1. it's just not implemented. And not because it would cause expensive queries.
2. range queries (<,>,<<,>>) are also not supported (yet) in xg